ga4gh/datamodel/datasets.py

```diff
@@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
     """
     variantsDirName = "variants"
     readsDirName = "reads"
+    featuresDirName = "sequenceAnnotations"
```

Comment:
Features is a type of record (discrete annotations) and there
will also be wiggle (continuous).  So is the dirname a module
or a record?

Jerome Kelleher <notifications@github.com> writes:
> In ga4gh/datamodel/datasets.py:
> 
> > @@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
> >      """
> >      variantsDirName = "variants"
> >      readsDirName = "reads"
> > +    featuresDirName = "sequenceAnnotations"
> 
> Should we call the directory features for consistency? This structure isn't
> really user-facing, so it might be good to be consistent about our naming.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub*
>